modules/mono/csharp_script.cpp

```diff
@@ -501,7 +501,8 @@ Vector<ScriptLanguage::StackInfo> CSharpLanguage::debug_get_current_stack_info()
 	_recursion_flag_ = true;
 	SCOPE_EXIT {
 		_recursion_flag_ = false;
-	};
+	}
+	FORCE_SEMICOLON
```

Comment:
Here's an example of the use of `FORCE_SEMICOLON`. Only use when the formatter is stubborn removing a necessary semi-colon.